docs/release-notes/1.11.0.md

```diff
@@ -2,6 +2,7 @@
 
 ```{rubric} Features
 ```
+* Support sparse chunks in dask {func}`~scanpy.pp.scale`, {func}`~scanpy.pp.filter_cells`, {func}`~scanpy.pp.filter_genes`, {func}`~scanpy.pp.normalize_total` and {func}`~scanpy.pp.highly_variable_genes` (`seurat` and `cell-ranger` tested) {pr}`2856` {smaller}`ilan-gold`
```

Comment:
```suggestion
* Support sparse chunks in dask {func}`~scanpy.pp.scale`, {func}`~scanpy.pp.normalize_total` and {func}`~scanpy.pp.highly_variable_genes` (`seurat` and `cell-ranger` tested) {pr}`2856` {smaller}`ilan-gold`
```

Should we remove those given our experiences today?

Reply:
Also it's possible that the sparse version will not have this problem so that note may be correct

Reply:
Let's remove then yes: https://github.com/scverse/anndata/pull/1432 just confirmed this solves the problem...so will depend on `anndata` release maybe then actually? If this goes in as patch release then this problem will solve itself...